src/Raven.Server/Documents/Handlers/Processors/Smuggler/AbstractSmugglerHandlerProcessorForImport.cs

```diff
@@ -108,6 +106,8 @@ await using (var stream = RequestHandler.GetDecompressedStream(section.Body, sec
                                     IgnoreDatabaseItemTypesIfCurrentVersionIsOlderThenClientVersion(context, ref blittableJson);
 
                                     options = JsonDeserializationServer.DatabaseSmugglerOptions(blittableJson);
+                                    options.AuthorizationStatus = GetAuthorizationStatus(RequestHandler.DatabaseName);
```

Comment:
Maybe better to have a constructor like `DatabaseSmugglerOptionsServerSide(DatabaseSmugglerOptionsServerSide options, AuthorizationStatus authStatus)` and avoid setting it via property completely

Reply:
We'll encounter an exception if we attempt to use AuthorizationStatus without setting it, which can only happen when using the deserializer.

Reply:
OK